tests/test_integration.py

```
@@ -0,0 +1,560 @@
+    if not fixture_doc.exists():
+        pytest.skip("CLI sample fixture not available")
+
+    input_doc = tmp_path / "smoke_input.docx"
+    shutil.copy(fixture_doc, input_doc)
+    output_doc = tmp_path / "smoke_output.docx"
+
+    monkeypatch.setenv("OPENAI_API_KEY", "test-key")
+    monkeypatch.chdir(tmp_path)
+
+    # Mock translation similar to CLI test
+    async def fake_translate(*args, **kwargs):
```

Ruff (0.13.1)

Unused function argument: `args`

(ARG001)